### DISCUSSION TITLE
generate QR code for hyper slow ( 13 )

### DISCUSSION THREAD

**Discussion by JiDey0007** [gopro/labs#1138] [Q&A]
In the generator I can't find the functionality to generate the slomotion in 400fps and 360pfs (plus dive)

This function deserves a QR code to be quickly started and go over to another configuration

What did I miss?

Another thing the WARM=x adds red stupidly or is it according to the progressive loss?

Thank you guys

**Comment by dnewman-gpsw** [comment]
mVB - mode video burst.  Currently no control to select which res and fps in slowmo burst.

WARM adds linear light red gain, this is the mathematically correct way to do it.

**Comment by JiDey0007** [comment]
Thank you for your answer

Too bad, the QR code goes so fast, I hope it will be the case later

For the red too bad, the gain could have been made on the proportion of blue increasing

I'm afraid of having too red images on the surface

We are going to test...

**Comment by JiDey0007** [comment]
I don't think it's relevant to make a post for that, just to know on the 13 we can activate Dive and warm but: that it lines to check that it is active on the Metadata reader?https://gopro.github.io/labs/control/metadata/

**Reply by dnewman-gpsw** [reply]
All under User Metadata 
![image](https://github.com/user-attachments/assets/f5a84116-19df-4458-b3d8-bea1e1033fae)

**Reply by JiDey0007** [reply]
Thanks

For Dive ok, I don't see Warm Cold 

Probably failed to do the code well
